chore: moves Cypress tests to a separate folder

gutoslv opened the pull request:
### Summary
Moves Cypress tests to a separate folder in order to use different dependencies than the main project
### Will affect visual aspect of the product
<!-- It includes visual changes? -->
NO

### Screenshots <!-- if applicable -->

### Test instructions
<!-- Describe how this pull request can be tested. -->

- Tests should pass into pipeline
- 

<!-- Issues that this pull request closes. -->
Closes  #2665.
<!-- Should look like this: `Closes #1, #2, #3.` . -->

selul commented:
@gutoslv only one small thing, why we have removed the [--frozen-lockfile](https://classic.yarnpkg.com/en/docs/cli/install/#toc-yarn-install-frozen-lockfile) flag ? Keeping it without it will update the dependencies, being prone to inconsistent results

gutoslv commented:
> @gutoslv only one small thing, why we have removed the [--frozen-lockfile](https://classic.yarnpkg.com/en/docs/cli/install/#toc-yarn-install-frozen-lockfile) flag ? Keeping it without it will update the dependencies, being prone to inconsistent results

I've removed them because it was needed for updating the new package.json, I've forgotten to revert the commits that changed it. I've just updated it with the --frozen-lockfile config.
